src/deroff.rs

```diff
@@ -1157,7 +1159,18 @@ fn deroff_files(files: &[String]) -> io::Result<()> {
             let mut decoder = GzDecoder::new(file);
             decoder.read_to_string(&mut string)?;
         } else {
-            file.read_to_string(&mut string)?;
+            match file.read_to_string(&mut string) {
+                Err(_) => {
+                    // TODO: This is a _bad_ workaround for latin1, we need to correctly decode input files
+                    let mut bytes = Vec::new();
+                    file.read_to_end(&mut bytes)?;
+
+                    for byte in bytes {
+                        string.push(byte as char);
+                    }
+                }
+                _ => {}
```

Comment:
```suggestion
                Err(e) if e.kind() == io::ErrorKind::InvalidData => { // not valid UTF-8
                    // TODO: This is a _bad_ workaround for latin1, we need to correctly decode input files
                    let mut bytes = Vec::new();
                    file.read_to_end(&mut bytes)?;

                    for byte in bytes {
                        string.push(byte as char);
                    }
                }
                Err(e) => {
                    eprintln!("unknown error {:?}", e);
                    continue;
                }
                Ok(_) => {}
```
I think we should only handle `InvalidData` here. I think we should import `ErrorKind` and remove `io::`. Also, it would be easier if we work on byte level so we don't need to care if it's valid utf-8 or not.